resources/views/navigation/nav.blade.php

```diff
@@ -0,0 +1,54 @@
+<nav class="lg:hidden">
+    <x-rapidez-statamic::nav-layer
+        id="navigation"
+        is-form
+        :children="Statamic::tag('nav:main')->fetch()"
+    />
+</nav>
+@php
+    $baseUrl = \Statamic\Facades\Site::current()->absoluteUrl();
+@endphp
+<nav class="relative border-y max-lg:hidden">
+    <ul class="flex items-center justify-center gap-8 font-semibold text-neutral [&>:not(:hover)]:hover:text-inactive text-sm">
+        @foreach (Statamic::tag('nav:main')->fetch() as $item)
+            @php
+                $level1Url = isset($item['linked_category']) && ($item['linked_category']?->value()['url_path'] ?? false) ? $baseUrl . '/' . $item['linked_category']->value()['url_path'] : $item['url'] ?? '';
```

Comment:
I see this check done multiple times, maybe we can add a helper function for it or something? Maybe like these: [https://github.com/rapidez/core/blob/master/src/helpers.php](https://github.com/rapidez/core/blob/master/src/helpers.php) ? Or even nicer maybe add a facade `RapidezStatamic::getNavUrl($item)` like we have for [the config](https://github.com/rapidez/core/blob/master/src/Rapidez.php#L42)

Reply:
Also as a addition i don't really like the level1, level2, level3 etc, can't you just call each iteration `$item` ? instead of; 
```
@foreach ($item['children'] as $level2)
```
do
```
@foreach ($item['children'] as $item)
```